scrapers/errata.py

```diff
@@ -198,3 +216,34 @@ def get_attached_bugs(self, advisory_id):
         """.format(advisory_id)
         log.info('Getting Bugzilla bugs tied to the advisory with ID {0}'.format(advisory_id))
         return self.teiid.query(sql)
+
+    def get_product_version_info(self, product_id):
```

Comment:
Could we merge this query into another join in `get_advisories`? If not, this is okay.

Reply:
Not directly, I think we might have to use nested queries to do that